modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/filename/FileTreeTestUtils.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache.persistence.filename;
+
+import java.util.regex.Pattern;
+
+import static org.apache.ignite.internal.processors.cache.persistence.filename.NodeFileTree.PART_FILE_PREFIX;
+
+/** */
+public class FileTreeTestUtils {
+    /**
+     * @return Pattern to find partition files.
+     */
+    public static Pattern partitionFilePattern() {
```

Comment:
The `partitionFilePattern` method is used only in one test - `IgnitePdsCheckpointRecoveryTest`. Lets move to it?